solver_spec.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'solver'
+
+describe Solver do
+  solver = Solver.new
+  context 'Getting factorial of a number (n):' do
+    it 'Raise an exception for factorial of negative numbers ie all numbers less than 0' do
+      expect(solver.factorial(-1)).to eq 'Number is less than 0'
+    end
+
+    it 'Return the factorial of 0 as 1' do
```

Comment:
- [ ] `required`
- [ ] amazing job so far! 👏 
- [ ] Good job with the test, however some of your tests are failing please fix them.
- [ ] And please see the screenshot for the failing tests 🥂 

![image](https://user-images.githubusercontent.com/66230306/202873364-aeef67e2-cf6f-4200-8613-b0b128b85e12.png)